pkg/backend/aws/ecs.go

```diff
@@ -369,6 +360,33 @@ func (ab *Backend) getLogEventsForTask(
 		return errors.Wrap(err, "could not describe task definition")
 	}
 
+	ch := make(chan *ecs.DescribeTasksOutput, 1)
+	go func() {
```

Comment:
We need to kill this goroutine if the timeout hits

Reply:
Its possible we don't need this goroutine. You could move this logic to line 385 and wrap the select statement in a for loop